free-2.md

```diff
@@ -58,12 +169,38 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think this might actually be a fairly feasible project. It looks like Paul
+Dapolito took a crack at it last year, but only targeted problem sets and
+memos. I think it would be good to take a more general approach, and that this
+would actually make for a very good DSLs project.
+
+I imagine most of the time would be spent learning a lot about the way LaTeX
+works and then deciding what parts of it are acceptable to cut out for the sake
+of nicer syntax and more readable code (I can see verbatim being problematic).
+
+I imagine that the implementation side would actually end up being very simple;
+you would just write some sort of preprocessor which expands the code into
+actual LaTeX.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+I think that this project could be varyingly ambitious. Making a whitespace
+sensitive version of LaTeX could be very challenging, but I imagine that by
+trimming down parts of the language you could make a more feasible project.
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+LaTeX is widely used and very well established, which I see as both a benefit
+and drawback. On the plus side, even small improvements can end up having a big
+impact because of the size of the user base. On the downside it is very well
+established, so users may be unwilling to switch over. Making it
+backwards-compatible would help with this though.
+
+This wouldn't exactly be an internal DSL, but the desire to make it at least
+somewhat compatible with LaTeX would make this project similar to creating an
+internal DSL. I think this would actually be a very positive experience because
+it requires you to think about two programming languages simultaneously, and
+try to find a way to weave them together.
```

Comment:
When we discussed it recently, I think we came to the conclusion that--as stated--this project puts a lot of effort on: (a) parsing whitespace and (b) understanding how LaTeX works. I think these are interesting...challenges...but the solution to both problems is essentially: "hacks" -- lots of semi-stapled-together code to handle all the edge cases.

You could get around the need to understand LaTeX's layout engine by having your language compile to LaTeX. That leaves you with whitespace parsing. If you wanted to go forward with this project, I would support it, but reluctantly :). That's because I think the experience will be less interesting and rewarding to you than the experience with geometric sketches...